usersimcrs/items/item_collection.py

```diff
@@ -50,46 +52,39 @@ def add_item(self, item: Item) -> None:
         self._items[item.id] = item
 
     def load_items_csv(
```

Comment:
Unfortunately, we need to deal with the case of genres, which are provided as a list of comma-separated values.
It might be better to have a mapping dict as an argument which could take additional info needed for post-processing, e.g.:
```
mapping = {
  "movieId": {"slot": "ID"},
  "title": {"slot": "TITLE"},
  "genres": {"slot": "GENRE", "multi-valued": True, "split_character": "|"},
  "keywords": {"slot": "KEYWORD", "multi-valued": True, "split_character": "|"},
}
```

It'd be important to have a test for this method.

We should also create a future TODO for loading additional slots (actors, directors) from other files.

Reply:
I added a mapping parameter to the method and updated as required.